### PR TITLE
Sdht 29 dead link report wbm fix

### PR DESCRIPTION
**Pull request by kbielawiec**
https://achieveinternet.atlassian.net/browse/SDHT-29

### CODE REVIEW

File: modules/custom/scripps_broken_links/src/Form/LinkCheckerConfig.php [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+      '#title' => t('Interval to check content for broken links.'),
+      '#default_value' => $config->get('check_interval'),
+      '#options' => $period,
+      '#description' => t('How often content will be checked for broken links.'),
+    ];
+
+    return parent::buildForm($form, $form_state);
+  }
+
+  public function submitForm(array &$form, FormStateInterface $form_state) {
```
Comment (csevb10) [inline modules/custom/scripps_broken_links/src/Form/LinkCheckerConfig.php]: Method documentation, please.

File: modules/custom/scripps_broken_links/src/Form/LinkCheckerConfig.php [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  protected function getEditableConfigNames() {
+    return [
+      'scripps_broken_links.adminsettings',
+    ];
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() {
```
Comment (csevb10) [inline modules/custom/scripps_broken_links/src/Form/LinkCheckerConfig.php]: Can this be done via an annotation?

File: modules/custom/scripps_broken_links/src/Form/LinkCheckerConfig.php (hunk 1 above)
Comment (kbielawiec) [reply modules/custom/scripps_broken_links/src/Form/LinkCheckerConfig.php]: Fixed

File: modules/custom/scripps_broken_links/src/Form/LinkCheckerConfig.php (hunk 2 above)
Comment (kbielawiec) [reply modules/custom/scripps_broken_links/src/Form/LinkCheckerConfig.php]: The class requires a getFormId() function

**Pull request merged**